### PR TITLE
Noarch

### PR DESCRIPTION
**Pull request by ocefpaf**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,8 @@ source:
 
 build:
   script: python setup.py install --single-version-externally-managed --record record.txt
-  number: 0
+  number: 1
+  noarch: python
```
Comment (xhochy) [inline recipe/meta.yaml]: Interesting feature, indeed! Is this line enough for `conda-smithy` to render the feedstock correctly?

Comment (ocefpaf) [reply recipe/meta.yaml]: Yep. First you change that line and then re-render.
(But I am having trouble with the entrypoint now :unamused:)

File: recipe/meta.yaml [hunk 2]
```diff
@@ -5,23 +5,22 @@ package:
   version: {{ version }}
 
 source:
-  fn: PyScaffold-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/p/pyscaffold/PyScaffold-{{ version }}.tar.gz
   sha256: 8cc9dcbe2e4be48a0c25d5484ad0eea62ee0d490f82c44a5f7ca83bb8a2ca62f
 
 build:
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python -m pip install --no-deps .
```
Comment (isuruf) [inline recipe/meta.yaml]: I think you don't need this change. Only the change in Line 17

Comment (ocefpaf) [reply recipe/meta.yaml]: This is borrowed from `ipython-feedstock` I had bad experience with some entrypoints when `pip` was not used. I'll take a look at the source if the entry point change and fix `17` later, but #9 worked without any modification :confused:

**Comment by ocefpaf** [comment]
@msarahan here is another data point where the entrypoints are failing with `noarch`.
Let me know if you want me to test anything.

**Pull request closed without merging**